resources/testharness.js

```diff
@@ -619,6 +645,20 @@
         return test_obj;
     }
 
+    /**
+     * Create a promise test.
+     *
+     * Promise tests are tests which are represeted by a promise
+     * object. If the promise is fulfilled the test passes, if it's
+     * rejected with an :js:class:`AssertionError` the test fails,
```

Comment:
This isn't true is it? Doesn't a promise test just fail if rejected with any value, not just `AssertionError`?

Reply:
Correct, apparently I'd forgotten how this actually worked when it came to this part.